app/main.py

```diff
@@ -1,2 +1,53 @@
+from typing import Any
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.length: int = 0
```

Comment:
Annotations for the attributes are not really necessary here. Type hints are already clear from the assigned values. You can skip annotations in such cases to write less code -  `LiudmylaKulinchenko`